pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_HDAWG8.py

```diff
@@ -0,0 +1,102 @@
+import time
+import logging
+import os
+from . import zishell_NH as zs
+from qcodes.utils import validators as vals
+from .ZI_base_instrument import ZI_base_instrument
+from qcodes.instrument.parameter import ManualParameter
+
+
+class ZI_HDAWG8(ZI_base_instrument):
+    """
+    This is PycQED/QCoDeS driver driver for the ZI_HDAWG8.
```

Comment:
Good point. In my opinion we should change the name to HDAWG. It is now a merge between the code name AWG8 and the name featured on the box. HD stands for High definition I suppose.